config-xml/tests/src/test/resources/query-handler.rhamt.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<test xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="query-handler-test-1">
+
+    <graph-query discriminator="FileResource" from="test-from" as="test-as">
```

Comment:
I miss the xsd definition for the graph-query.

Reply:
Fixed.